### DISCUSSION TITLE
在word、powerpoint、onenote等微软办公软件里出问题了

### DISCUSSION THREAD

**Discussion by coisini-95** [shuax/MouseInc.Settings#369] [Q&A]
在word、powerpoint、onenote这几个微软办公软件里选中一段文字后，原本在文字的右上角会出现字体字号设置的框，但是开启了mouselnc后那个框会一闪而过，没法出现，也没法点，这是咋回事呢，是bug吗？

**Comment by shuax** [comment]
![image](https://user-images.githubusercontent.com/6940583/188545927-f7ed8506-0b35-4b4c-b738-5565aca0b3ca.png)
我正常

**Reply by coisini-95** [reply]
我发现了，是那个快速复制设置的缘故，选中直接复制会导致右上角的框不能出现，感谢感谢
